exchange/exchange.go

```diff
@@ -325,7 +326,7 @@ func (e *exchange) HoldAuction(ctx context.Context, r *AuctionRequest, debugLog
 
 	// If we need to cache bids, then it will take some time to call prebid cache.
 	// We should reduce the amount of time the bidders have, to compensate.
-	auctionCtx, cancel := e.makeAuctionContext(ctx, cacheInstructions.cacheBids)
+	auctionCtx, cancel := e.makeAuctionContext(ctx, cacheInstructions.cacheBids, &r.TmaxAdjustments)
```

Comment:
should update comments line to also highlight `tmaxAdjustment`.  
```
// If we need to cache bids, then it will take some time to call prebid cache.
// Similarly, PBS will need some time to prepare auction response.
// We should reduce the amount of time the bidders have, to compensate.
```